.github/workflows/milestone.yml

```diff
@@ -12,9 +12,7 @@ jobs:
       - name: Checkout code
```

Comment:
`Remove obsolete hotfix` -> this is very generic title, can you please have a bit more specific title?

Reply:
Do you mean `commit message` or the `title` itself of the `commit message`? 
If first, the `title` is described in the `body` of the `commit message`.
if second - it's hard for me to find better laconic title.

Reply:
`Do not display maven version` this is `how` it was done, but not `what`.

Reply:
I've applied your comment.